src/Serilog.Sinks.Elasticsearch/Serilog.Sinks.ElasticSearch.Symbols.nuspec

```diff
@@ -11,8 +11,8 @@
     <iconUrl>http://serilog.net/images/serilog-sink-nuget.png</iconUrl>
     <tags>serilog logging elasticsearch</tags>
     <dependencies>
-      <dependency id="Serilog" version="[1.5.14,2)" />
-      <dependency id="Elasticsearch.Net" version="2.0.4" />
+      <dependency id="Serilog" version="2.0" />
+      <dependency id="Elasticsearch.Net" version="2.3.3" />
```

Comment:
Not sure how the symbols package is being built, but the dependencies mentioned in regard to the base package (below) probably need to be specified here as well.